API.md

```diff
@@ -129,7 +129,8 @@ instead return a [Joi ValidationError object](https://joi.dev/api/?v=17.9.1#vali
       createdDt: string (date),
       updatedDt: string (date),
     },
-    stewardAccountId: string,
+    stewardEmail: string,
+    stewardName: string,
```

Comment:
If we're returning multiple properties of the steward, I think we should make the steward its own object within our directive object.
```
steward: {
  email: string,
  name: string,
}
```

Reply:
Of course that probably breaks `web-app`'s integration with this endpoint, which isn't ideal.

Reply:
That makes sense - @meisekimiu let's discuss in planning.